actors/evm/src/lib.rs

```diff
@@ -125,7 +125,8 @@ impl EvmContractActor {
             Outcome::Revert => Err(ActorError::unchecked_with_data(
                 EVM_CONTRACT_REVERTED,
                 "constructor reverted".to_string(),
-                RawBytes::serialize(BytesSer(&output.return_data)).unwrap(),
+                // TODO: Can we do better?
```

Comment:
We could, but we'll lose the codec in the receipt. It's not the end of the world (we can just "know" the codec), but it makes me a bit unhappy.

I'd leave this as it is for now, and consider changing in a followup patch.